easybuild/easyconfigs/i/Imath/Imath-3.1.5-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'Imath'
+version = '3.1.5'
+
+homepage = 'https://imath.readthedocs.io/en/latest/'
+description = """
+Imath is a C++ and python library of 2D and 3D vector, matrix, and math operations for computer graphics
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+sources = [{
+    'filename': '%(name)s_v%(version)s.tar.gz',
```

Comment:
This causes a git clone to be done, which results in a non-reproducible checksum. Can't we just download an archive from `https://github.com/AcademySoftwareFoundation/%(namelower)s/archive/` or something, like for OpenCV?

Reply:
Fixed it myself